docs/go-crashcourse.md

```diff
@@ -130,6 +130,28 @@ Note: Try to get a percentage score
 
 ## Dependency management
 
+Install glide:
+```bash
+brew install glide
+```
+
+Initialize your repo and fetch the dependencies:
+```bash
+glide init
+glide update
+```
+
+You can add the `vendor/` folder to your `.gitignore` file.
```

Comment:
I would assume that like with working with npm, it's a best practice to always have the dependency directory in the .gitignore. Perhaps, we should say 'you should' here to stress that, and add a reason why, something like:

```
You should add the `vendor/` folder to your `.gitignore` file. It will help keep your repo small and all of your dependencies are already well defined in `glide.yaml`.
```

Reply:
Addressed.